packages/core/services/api.js

```diff
@@ -42,41 +45,37 @@ module.exports = {
     async authenticate(ctx, route, req, res) {
       if (req.headers.signature) {
         return ctx.call('signature.authenticate', { route, req, res });
-      } else {
-        const token = req.headers.authorization?.split(' ')[1];
-        if (token) {
-          const payload = await ctx.call('auth.jwt.decodeToken', { token });
-          if (payload?.azp) {
-            // This is a OIDC provider-generated ID token
-            return ctx.call('oidc-provider.authenticate', { route, req, res });
-          } else {
-            // Otherwise it is a custom JWT token (used by ActivityPods frontend) or a capability URL
-            return ctx.call('auth.authenticate', { route, req, res });
-          }
+      }
+      const token = req.headers.authorization?.split(' ')[1];
+      if (token) {
+        const payload = await ctx.call('auth.jwt.decodeToken', { token });
+        if (payload?.azp) {
+          // This is a OIDC provider-generated ID token
+          return ctx.call('oidc-provider.authenticate', { route, req, res });
         }
+        // Otherwise it is a custom JWT token (used by ActivityPods frontend) or a capability URL
+        return ctx.call('auth.authenticate', { route, req, res });
       }
+
       ctx.meta.webId = 'anon';
       return null;
     },
     async authorize(ctx, route, req, res) {
       if (req.headers.signature) {
         return ctx.call('signature.authorize', { route, req, res });
-      } else {
-        const token = req.headers.authorization?.split(' ')[1];
-        if (token) {
-          const payload = await ctx.call('auth.jwt.decodeToken', { token });
-          if (payload.azp) {
-            // This is a OIDC provider-generated ID token
-            return ctx.call('oidc-provider.authorize', { route, req, res });
-          } else {
-            // Otherwise it is a custom JWT token (used by ActivityPods frontend)
-            return ctx.call('auth.authorize', { route, req, res });
-          }
-        } else {
-          ctx.meta.webId = 'anon';
-          throw new E.UnAuthorizedError(E.ERR_NO_TOKEN);
+      }
+      const token = req.headers.authorization?.split(' ')[1];
+      if (token) {
+        const payload = await ctx.call('auth.jwt.decodeToken', { token });
+        if (payload.azp) {
+          // This is a OIDC provider-generated ID token
+          return ctx.call('oidc-provider.authorize', { route, req, res });
         }
+        // Otherwise it is a custom JWT token (used by ActivityPods frontend)
+        return ctx.call('auth.authorize', { route, req, res });
       }
+      ctx.meta.webId = 'anon';
+      throw new E.UnAuthorizedError(E.ERR_NO_TOKEN);
```

Comment:
Personnaly I think the `no-else-return` ESlint rule make the code more difficult to read. The `else` keyword helps to see what is happening. That's why I disabled it on other repos. Would you agree with that ?

Reply:
I see, I don't have a strong opinion on this. In many cases I find code easier to read because it reduces the amount of indent but it's also a question of context and enforcing this with eslint is a bit overkill :)